### PR TITLE
Fix graph pagination

### PR DESCRIPTION
**Pull request by KN4CK3R**
Pagination is moved to the side on long graphs.

grafik

### CODE REVIEW

File: templates/repo/graph/commits.tmpl [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		{{ range $commitI, $commit := .Graph.Commits }}
 			<li id="commit-{{$commit.Rev}}" data-flow="{{$commit.Flow}}">
```
[the remark this answers is not in the document]
Comment (zeripath) [reply templates/repo/graph/commits.tmpl]: yeah do that.

[the remark this answers is not in the document]
Comment (zeripath) [reply templates/repo/graph/commits.tmpl]:
```suggestion
			<li {{if $commit.Rev}}id="commit-{{$commit.Rev}}"{{end}} data-flow="{{$commit.Flow}}">
```

**Review by zeripath — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by techknowlogick** [comment]
🚀

**Comment by techknowlogick** [comment]
@KN4CK3R please send backport to release/v1.14 :)

**Pull request merged**